cmd/svcat/binding/unbind_cmd.go

```diff
@@ -82,6 +118,33 @@ func (c *unbindCmd) deleteBinding() error {
 
 func (c *unbindCmd) unbindInstance() error {
 	bindings, err := c.App.Unbind(c.Namespace, c.instanceName)
-	output.WriteDeletedBindingNames(c.Output, bindings)
-	return err
+	if err != nil {
+		return err
+	}
+
+	if c.wait {
+		glog.V(2).Infof("Waiting for the bindings to be deleted...")
+		pollInterval := 1 * time.Second
```

Comment:
I would recommend making a flag to be able to set the poll interval and the default can still be 1 second. A 1 second poll delay in a large script could mean it runs significantly slower if the commands finish in 1001 milliseconds